### PR TITLE
in blackest night

### PR DESCRIPTION
**Pull request by Wallace-A**
Please double check all these things work:
added a create costume bar to the navbar for testing. Goes to costumes/new
added a manage bookings buton to the nav bar as a test. Goes to /bookings
user can create a costume
user can upload a photo with costume
creating a costume sends the user to the show page
stored photos now display on the show page
Screenshot from 2021-02-23 21-29-54
Screenshot from 2021-02-23 21-31-08

### CODE REVIEW

**Comment by blamb888** [comment]
Dude, Andy you are the bomb diggity man. This looks really good. How do I pull this branch to test it before I push to master again?

File: app/controllers/costumes_controller.rb [hunk 1]
```diff
@@ -38,6 +38,6 @@ def set_costume
   end
 
   def costume_params
-    params.require(:costume).permit(:name, :user, :category, :description, :price)
+    params.require(:costume).permit(:name, :user, :category, :description, :price, :photo)
```
Comment (blamb888) [inline app/controllers/costumes_controller.rb:41]: Looks good.

File: app/views/costumes/_costume_show_card.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
 <div class="card-trip">
-  <img src="https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/greece.jpg" />
+  <%= cl_image_tag costume.photo.key,crop: :fill  %>
```
Comment (blamb888) [inline app/views/costumes/_costume_show_card.html.erb]: perfect upgrade!

**Comment by Wallace-A** [comment]
I want to say git checkout branch_name

**Comment by blamb888** [comment]
image
Trying lol

**Comment by Wallace-A** [comment]
weird. I just did it on you costume-images-on-booking-cards one

**Comment by Wallace-A** [comment]
wait you did it. the branch changed

**Review by blamb888 — APPROVED** [review]
Looks great. Wish I could figure out how to pull it to my local to test. But seriously looks great. We might have a second user story for tomorrow's pitch!

**Comment by blamb888** [comment]
yeah, the branch changed but it wouldn't let me pull? My local host didn't change either:(

**Comment by Wallace-A** [comment]
No that's fine. if you're in the branch then you pulled it I think

**Comment by blamb888** [comment]
I figured it out. I, like the dum dum I am, accidentally created a different branch:( But I managed to co yours and pull. Checking it out in localhost rn

**Comment by blamb888** [comment]
image
???

**Comment by Wallace-A** [comment]
Now i've got the same problem you had. :(

**Comment by blamb888** [comment]
I don't know why but I keep getting this undefined method for nil class error pointing back to the controller.
image
The create looks good, though. So I dunno.

**Comment by blamb888** [comment]
Fudge, I'm a dum dum again! Bundle install saves the day lol
image

**Comment by Wallace-A** [comment]
Noice!

**Comment by blamb888** [comment]
Noice indeed. Anyway we can get that image centered?

**Comment by blamb888** [comment]
oh dip maybe it is? The image was pretty tall so maybe that's the center?
spawn

**Comment by Wallace-A** [comment]
yeah, they're centered but too big.

**Comment by blamb888** [comment]
image
`  <%= cl_image_tag costume.photo.key, width: 350, height: 300, gravity: :face, crop: :fill %>
`

**Comment by blamb888** [comment]
by our powers combined we are PHOTO CARDS!

**Comment by Wallace-A** [comment]
Fucking yes mate! Maybe save that for a separate commit.

**Comment by blamb888** [comment]
Whattya mean? Like make another branch and add it to our other image tags?

**Comment by blamb888** [comment]
Well I pushed it I think to this branch. But I only added it to the _costume_show_card partial.

**Comment by blamb888** [comment]
I'm going to go ahead and merge your branch now. Fucking awesome work @Wallace-A

**Pull request merged**